pkg/operator/certrotation/target.go

```diff
@@ -50,6 +50,9 @@ type RotatedSelfSignedCertKeySecret struct {
 	// rotation on expiration only, but not interfere with the ordinary rotation controller.
 	RefreshOnlyWhenExpired bool
 
+	// Owner is an optional reference to add to the secret that this rotator creates.
+	Owner *metav1.OwnerReference
```

Comment:
Who is the typical owner, then? Why would you want the secret to be garbage collected if it disappears?

Please explain in the godoc or remove.